code/__DEFINES/lights.dm

```diff
@@ -1,5 +1,5 @@
 ///How much power emergency lights will consume per tick
-#define LIGHT_EMERGENCY_POWER_USE 0.2
+#define LIGHT_EMERGENCY_ENERGY_USE (0.2 KILO JOULES)
```

Comment:
Optional but you will get bonus points if you do; convert this back to power and make the light power usage scale by seconds_per_tick.